pkg/notifications/new_discussion_added_meta.go

```diff
@@ -0,0 +1,45 @@
+package notifications
+
+import (
+	"context"
+
+	"github.com/google/uuid"
+
+	"github.com/cms-enterprise/mint-app/pkg/models"
+	"github.com/cms-enterprise/mint-app/pkg/sqlutils"
+)
+
+// ActivityNewDiscussionAddedCreate creates an activity for when a Discussion is added.
+func ActivityNewDiscussionAddedCreate(
```

Comment:
I think looking at the dates changed activity could be good here. I think also makes sense to potentially move the email logic to happen here instead of in resolvers.